packages/chain-adapters/src/types.ts

```diff
@@ -316,3 +316,15 @@ export enum ChainAdapterDisplayName {
   Dogecoin = 'Dogecoin',
   Litecoin = 'Litecoin',
 }
+
+export type BroadcastTransactionInput = {
+  senderAddress: string
+  receiverAddress: string | undefined // this is not defined for staking etc
```

Comment:
```suggestion
  receiverAddress: string | undefined // this is not defined for Cosmos SDK native staking etc
```

Reply:
Actually correct for now, given the intent is for contracts calls not to have a `receiverAddress`

Reply:
To be revisited if/when we implement verification for contracts